mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -50,16 +48,15 @@ private object DTO {
   }
 
   case class QuotaUsageUpdatedEvent(user: User, quotaRoot: QuotaRoot, countQuota: Quota[QuotaCount],
-                                    sizeQuota: Quota[QuotaSize], time: Instant) extends QuotaEvent {
-    override def getQuotaRoot: QuotaRoot = quotaRoot
+                                    sizeQuota: Quota[QuotaSize], time: Instant) extends Event {
+    def getQuotaRoot: QuotaRoot = quotaRoot
 
     override def toJava: JavaQuotaEvent =
       new JavaQuotaUsageUpdatedEvent(user, getQuotaRoot, countQuota.toJava, sizeQuota.toJava, time)
   }
-
 }
 
-private object JsonSerialize {
+private class JsonSerialize(mailboxIdFactory: MailboxId.Factory) {
```

Comment:
I don't see the place you use this `mailboxIdFactory`

Reply:
Used later in an other PR.

Here I intend to solve future conflicts to ease work parallelization in this Sprint.

(If I did not added this MailboxId.Factory then I can not justify going from object to class, thus even if it is not used, I believe it is important)